### DISCUSSION TITLE
Title of the tab and the table of a relation manager

### DISCUSSION THREAD

**Discussion by jerry-whoami** [filamentphp/filament#8145] [Help] [answered]
Hi,
How can you change the label of the tabs and the title of a relation manager?
![image](https://github.com/filamentphp/filament/assets/95326190/e1919027-3879-4060-aa5f-376cb49d61ee)
Thx

**Comment by leandrocfe** [comment, marked as answer]
```php
protected static ?string $title = 'tab title';

--

$table
...
->heading('table title')
```

**Reply by qodlizaka** [reply]
What if I want to use translation for the `$title` attribute? As it is a static attribute.

**Reply by leandrocfe** [reply]
`getTitle()` method?

**Reply by youcefkacemDev** [reply]
how to do that is not working 

    protected static ?string getTitle():string {
        return __('netConnect.Companies');
    }

**Reply by maniflames** [reply]
For anyone bumping into this that needs to support localization:

```php
public static function getTitle(Model $ownerRecord, string $pageClass): string
{
    return __('tab title');
}
```

**Reply by abenzakour** [reply]
> For anyone bumping into this that needs to support localization:
> 
> ```
> public static function getTitle(Model $ownerRecord, string $pageClass): string
> {
>     return __('tab title');
> }
> ```

dont forget to add : `use Illuminate\Database\Eloquent\Model;`
